kuronometer-core/src/main/scala/com/github/pedrovgs/kuronometer/free/interpreter/api/KuronometerApiClientConfig.scala

```diff
@@ -0,0 +1,10 @@
+package com.github.pedrovgs.kuronometer.free.interpreter.api
+
+case class KuronometerApiClientConfig(
+                                       scheme: String = "https",
```

Comment:
Should raise it as an issue. With Kotlin raising I expect more people trying to use gradle with Scala